algoliasearch-apache/pom.xml

```diff
@@ -0,0 +1,51 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <parent>
+        <artifactId>algoliasearch</artifactId>
+        <groupId>com.algolia</groupId>
+        <version>3.0.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>algoliasearch-apache</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <async-http-client.version>4.1.4</async-http-client.version>
+        <jsr305.version>3.0.2</jsr305.version>
+        <algoliasearch-core-version>3.0.0-SNAPSHOT</algoliasearch-core-version>
+    </properties>
+
+    <dependencies>
+        <!--Apache http client-->
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpasyncclient</artifactId>
+            <version>${async-http-client.version}</version>
```

Comment:
I am not sure we really need a property for this version. We do not plan to allow its overriding on the command line, and the value is used only once (here).

Reply:
Fixed!